source/_integrations/event.markdown

```diff
@@ -0,0 +1,82 @@
+---
+title: Event
+description: Instructions on how to use event entities in Home Assistant.
+ha_category:
+  - Events
+ha_release: 2023.8
+ha_quality_scale: internal
+ha_domain: event
+ha_codeowners:
+  - '@home-assistant/core'
+ha_integration_type: entity
+---
+
+Events are signals that are emitted when something happens, for example, when a user presses a physical button like a doorbell or when a button on a remote control is pressed.
+
+These events are stateless. For example, a doorbell does not have a state like being "on" or "off" but instead is momentarily pressed. Some events can have variations in the type of event that is emitted. For example, maybe your remote control is capable of emitting a single press, a double press, or a long press.
+
+The event entity can capture these events in the physical world and makes them available in Home Assistant as an entity.
+
+Please note the event integration cannot be directly used; you cannot create your own event entities using this integration. This integration is a building block for other integrations to use, enbaling them to create event entities for you.
+
+## The state of an event entity
+
+The event entity is stateless, as in, it cannot have a state like the `on` or `off` state that, for example, a normal switch or light entity has.
+
+Therefore, every event entity keeps track of the timestamp of when the last time the emitted event has been detected.
+
+Because the state of an event entity in Home Assistant is a timestamp, it means we can use it in our automations. For example:
+
+```yaml
+trigger:
+  - platform: state
+    entity_id: event.doorbell
+action:
+  - service: notify.frenck
+    data:
+      message: "Ding Dong! Someone is at the door!"
+```
+
+## Event types
+
+Besides the timestamp of the last event, the event entity also keeps track of the last event type that has been emitted. This can be used in automations to trigger different actions based on the type of event.
```

Comment:
```suggestion
Besides the timestamp of the last event, the event entity also keeps track of the event type that has last been emitted. This can be used in automations to trigger different actions based on the type of event.
```